README.md

```diff
@@ -1,176 +1,98 @@
 
-For example, if you were implementing a custom resource for a Database, you might provide a DatabaseSpec like the following:
+### [minio.go](./cmd/minio.go)
 
-``` go
-type DatabaseSpec struct {
-	Databases []string `json:"databases"`
-	Users     []User   `json:"users"`
-	Version   string   `json:"version"`
-}
+Responsible for the configuration of MinIO buckets for the given `Profile`. Configured MinIO buckets may be used by the associated `Profile` for object storage accessible directly within their notebooks.
 
-type User struct {
-	Name     string `json:"name"`
-	Password string `json:"password"`
-}
-```
+### [network.go](./cmd/network.go)
 
-## Validation
+Responsible for the following networking policies:
 
-To validate custom resources, use the [`CustomResourceValidation`](https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/#validation) feature. Validation in the form of a [structured schema](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#specifying-a-structural-schema) is mandatory to be provided for `apiextensions.k8s.io/v1`.
+- Ingress from the ingress gateway
+- Ingress from knative-serving
+- Egress to the cluster local gateway
+- Egress from unclassified workloads
+- Egress from unclassified workloads to the ingress gateway
+- Egress to port 443 from protected-b workloads
+- Egress to the daaas-system
+- Egress to vault
+- Egress to pipelines
+- Egress to MinIO
+- Egress to Elasticsearch
+- Egress to Artifactory
 
-### Example
+### [notebook.go](./cmd/notebook.go)
 
-The schema in [`crd.yaml`](./artifacts/examples/crd.yaml) applies the following validation on the custom resource:
-`spec.replicas` must be an integer and must have a minimum value of 1 and a maximum value of 10.
+Responsible for the configuration of `Notebook` resources within Kubeflow. This controller adds a default option for running a protected-b `Notebook`.
 
-## Subresources
+### [quotas.go](./cmd/quotas.go)
 
-Custom Resources support `/status` and `/scale` [subresources](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#subresources). The `CustomResourceSubresources` feature is in GA from v1.16.
+Responsible for the generation of `ResourceQuotas` for a given profile. Management of `ResourceQuotas` is essential, as it provides the constraint for total amount of compute resources that are consumable within the `Profile`. Since the `ResourceQuotas` definition included in [quotas.go](./cmd/quotas.go) provides constraints for cpu and memory, the limits for the values must be defined. These limits are defined as `LimitRange` resources and are managed by [limitrange.go](./cmd/limitrange.go).
 
-### Example
+#### Quota Labels
```

Comment:
Link to this:

https://github.com/StatCan/aaw-kubeflow-profiles-controller/issues/16

Reply:
Do you mean I should replace ####Quota Labels with ####[Quota Labels](https://github.com/StatCan/aaw-kubeflow-profiles-controller/issues/16)?